crates/library/std/src/math.fe

```diff
@@ -0,0 +1,15 @@
+pub fn min(_ x: u256, _ y: u256) -> u256 {
```

Comment:
I just wanted to suggest that we could define an `Ord` trait and then have the `min` / `max` functions be generic and work with any `T` that is `Ord` but free floating functions such as these can not use generics yet as we only allow struct methods to be generic for now....so, nothing to see here.